src/archivematicaCommon/lib/env_configparser.py

```diff
@@ -0,0 +1,73 @@
+import os
+import ConfigParser
+import functools
+
+
+def fallback_option(fn):
```

Comment:
It doesn't. See my other comment above. I initially tried to support both py2 and py3 but the differences are important. Not sure if it's worth the time. Making this subclass py3-only would be very easy, but supporting both looked kind of a nightmare for a newcomer like me.